pkg/disttask/framework/taskexecutor/manager.go

```diff
@@ -270,18 +240,11 @@ func (m *Manager) handleExecutableTasks(tasks []*proto.Task) {
 		m.addHandlingTask(task.ID)
 		m.slotManager.alloc(task)
 		t := task
-		err = m.executorPool.Run(func() {
+		m.executorWG.Go(func() {
```

Comment:
Could we rename `Go` to `RunWithLog` or other names which is unify `WaitGroupWrapper`'s functions?

I don't think this function is used anywhere else, so I just mentioned it in this PR.

Reply:
https://github.com/pingcap/tidb/pull/50664/commits/e7228e463a86d03ff5399e35315897b9b1cd1d3b